src/wiki/plugins/macros/mdx/macro.py

```diff
@@ -205,29 +205,37 @@ def pspecial(self, **kwargs):
 
     def ss(self, **kwargs):
         args = argize_kwargs(kwargs)
-        cl = None
+
+        construal = None
```

Comment:
`cl` here was short for "class", not "construal". It's the "optional class" arg of the macro.

Reply:
`cls` is also used below (I don't recall why we needed separate variables)